tests/integration_test.go

```diff
@@ -0,0 +1,128 @@
+// +build integration
+
```

Comment:
Can we add a package-level comment describing what this test is for? At this point it would also be nice to have a small comment on the readme w/ a one-liner summary of the various tests and what they are for (swarm, recovery, integration)

Reply:
yep definitely